src/ResourceGraph/ResourceGraph/Utilities/TableExtensions.cs

```diff
@@ -67,7 +67,7 @@ public static IEnumerable<PSObject> ToPsObjects(this Table table)
                 {
                     var value = row[columnIndex];
                     var normalizedValue =
-                        (value as JObject)?.ToPsObject() ?? value;
+                        (value as JToken)?.ToPsObject() ?? value;
```

Comment:
@demyanenko if we have examples in the markdown files with the corresponding output from running the cmdlets, we should update the output to reflect the changes being made here

Reply:
It is a fairly corner case so the markdown examples are not covering it. I appreciate the concern.